tests/xapi-plugins/test_raid.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import pytest
+
+# Requirements:
+# From --hosts parameter:
+# - host(A1): first XCP-ng host > 8.2.
+
+@pytest.fixture(scope='session')
+def host_with_raid(host):
+    dummy_raid = False
+    try:
+        host.call_plugin('raid.py', 'check_raid_pool')
```

Comment:
I would prefer that you'd simply check that `/dev/md127` doesn't exist on the host.